### PR TITLE
Update channels api docs

### PR DESCRIPTION
**Pull request by SamantazFox**
Updated the docs to reflect the changes in the "channels" API from https://github.com/iv-org/invidious/pull/3419

~~Note: I haven't tried the PR locally, so IDK how the rendered Markdown looks like.~~

### CODE REVIEW

**Comment by SamantazFox** [comment]
I've tested the changes locally, they now look good!

File: mkdocs.yml [hunk 1]
```diff
@@ -52,3 +55,8 @@ extra:
 
 markdown_extensions:
   - pymdownx.magiclink
+  - pymdownx.highlight:
+      anchor_linenums: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences
```
Comment (TheFrenchGhosty) [inline mkdocs.yml:62]: Reason for this addition?

Comment (SamantazFox) [reply mkdocs.yml:62]: For syntax highlight, as per https://squidfunk.github.io/mkdocs-material/reference/code-blocks/

Comment (TheFrenchGhosty) [reply mkdocs.yml:62]: Alright, I'll assume you did your research and that they're good to add :)

**Pull request merged**